src/components/HealthSafetyPolicy.astro

```diff
@@ -0,0 +1,70 @@
+---
+
+---
+
+<style>
+    .content-wrapper {
+        max-width: var(--max-page-width);
+        margin-inline: auto;
+    }
+
+    h3 {
+        margin-block: 2.5rem 0rem;
+    }
+</style>
+<div class="content-wrapper">
+    <h2>Health, Safety and Accessibility</h2>
```

Comment:
That's a pretty rough entry, maybe we can add an introductory sentence that from a positive POV shows why these policies are important to us? i.e. make the conference enjoyable for everyone, etc.